Reply/app/src/main/java/com/materialstudies/reply/ui/nav/AccountViewHolder.kt

```diff
@@ -16,23 +16,23 @@
 
 package com.materialstudies.reply.ui.nav
 
+import androidx.compose.ui.platform.ComposeView
 import androidx.recyclerview.widget.RecyclerView
 import com.materialstudies.reply.data.Account
-import com.materialstudies.reply.databinding.AccountItemLayoutBinding
+import com.materialstudies.reply.ui.common.addAccountItem
 
 /**
  * ViewHolder for [AccountAdapter]. Holds a single account which can be selected.
  */
 class AccountViewHolder(
-    val binding: AccountItemLayoutBinding,
+    private val composeView: ComposeView,
     val listener: AccountAdapter.AccountAdapterListener
-) : RecyclerView.ViewHolder(binding.root) {
+) : RecyclerView.ViewHolder(composeView) {
 
     fun bind(accnt: Account) {
-        binding.run {
-            account = accnt
-            accountListener = listener
-            executePendingBindings()
-        }
+        composeView.addAccountItem(
```

Comment:
For example, in this case the view composition strategy should be the default (optimized for pooling container)